public/css/print.css

```diff
@@ -1,42 +1,78 @@
 body {
-	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+	font-family: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+	font-size: 10px;
+	height: 210mm;
+	width: 296mm;
+	margin-top: 11rem;
+	margin-left: -10rem;
+	-webkit-transform: rotate(-90deg);
+	-moz-transform:rotate(-90deg);
+	filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=3);
```

Comment:
wow! is that an IE specific "yes, you will rotate 3 steps, you recalcitrent browser" rule?